code/parachain/frame/staking-rewards/src/test/test_update_reward_pools.rs

```diff
@@ -1,7 +1,7 @@
 use composable_tests_helpers::test::{
```

Comment:
test/test_ and inside it [test]. not sure if need to say test so many times?

Reply:
I didn't write this original crate, I just went with the existing structure. I'm planning to restructure the `composable-tests-helpers` crate in the future though, since the current layout is definitely not ideal